helpers/database.js

```diff
@@ -180,13 +200,10 @@ function Migrator (pgp, db) {
  * @return {function} error|cb
  */
 module.exports.connect = function (config, logger, cb) {
-	var pgOptions = {
-		pgNative: true
-	};
-
-	var pgp = require('pg-promise')(pgOptions);
-	var monitor = require('pg-monitor');
-
+	try {
+		monitor.detach();
+	} catch (ex) {}
```

Comment:
Is this really ok? If we catch that error we should do something else.

Reply:
this will produce exception also if run for the first time - if attached multiple times it results in the warning. So in the first case, we would receive exception message but the behaviour will be correct